frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -45,20 +48,20 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     'cluster',
   );
   React.useEffect(() => {
-    watchK8sResource(cephClusterResource);
     watchK8sResource(SubscriptionResource);
+    watchK8sResource(ocsResource);
     return () => {
-      stopWatchK8sResource(cephClusterResource);
       stopWatchK8sResource(SubscriptionResource);
+      stopWatchK8sResource(ocsResource);
     };
   }, [watchK8sResource, stopWatchK8sResource]);
 
   const infrastructurePlatform = getInfrastructurePlatform(infrastructure);
-
-  const cephCluster = _.get(resources, 'ceph');
-  const cephClusterLoaded = _.get(cephCluster, 'loaded', false);
-  const cephClusterData = _.get(cephCluster, 'data') as K8sResourceKind[];
-  const cephClusterName = getName(_.get(cephClusterData, 0));
+  const ocs = resources?.ocs;
+  const ocsLoaded = ocs?.loaded || false;
+  const ocsError = ocs?.loadError;
+  const cluster = ocs?.data?.find((item: OCSStorageClusterKind) => item.status.phase !== 'Ignored');
```

Comment:
```suggestion
  const ocsData = ocs?.data as K8sResourceKind[];
  const cluster = ocsData?.find((item: OCSStorageClusterKind) => item.status.phase !== 'Ignored');
```

Reply:
Since we have a OCSStorageClusterKind I would prefer we use it instead of the general K8sResourceKind